addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1043,7 +1044,7 @@ const SelectUserValueWidget = BaseSelectionUserValueWidget.extend({
                 }
             }
         } else {
-            textContent = this.PLACEHOLDER_TEXT;
+            textContent = _t("Custom");
```

Comment:
Remind/Explain to me why this change was not about changing the value of `this.PLACEHOLDER_TEXT`, I don't remember/understand.

Reply:
The `PLACEHOLDER_TEXT` constant was added in [b7b05a2d1c1d5](https://github.com/odoo/odoo/commit/b7b05a2d1c1d5), around the time I reworked on it after your 1st review, and consistently changed the text from "/" to "None" wherever it was used. (* except on the line `if (this.menuTogglerEl.textContent === '/') {` where it's apparently been forgotten.)
To my understanding, the "Custom" text we agreed on shouldn't be used wherever the PLACEHOLDER_TEXT is set (e.g. togglers where no value is selected should remain "None" - see for instance when you drop a "Dynamic snippet": filter & template are "None" by default, this depends on the placeholder).

Reply:
@robinlej 

> * except on the line if (this.menuTogglerEl.textContent === '/') { where it's apparently been forgotten

Did you make a PR to fix that? :+1: 

> togglers where no value is selected should remain "None"

Well that's the point :wink: Your PR is changing exactly that by "Custom". I tested the dynamic snippet to see what you were talking about (I used the "Products" snippet), it's exactly broken like I thought it would:
![image](https://github.com/odoo/odoo/assets/10338094/7415140f-1f51-4803-a84f-ab91f4b2713c)
No tags is selected yet and it shows "Custom", it does not make sense :wink: I don't think we ever agreed that all select that use a non-existing value in the selection list shows "Custom": this change should probably not be about a generic change on the widget but a parameter used by your specific select only, no?